### ISSUE TITLE
Can't show some functions in a workspace

### ISSUE THREAD

**Issue by jrmuizel** [pacak/cargo-show-asm#41]
I was unable to get something like the following to work in [miniz_oxide](https://github.com/Frommi/miniz_oxide) git repo.
```
cargo asm  -p miniz_oxide --lib "<miniz_oxide::deflate::core::CompressorOxide as core::default::Default>::default"
```
When I copied the `miniz_oxide` subdirectory out of the workspace I was able to get it to work.

**Comment by jrmuizel** [comment]
Yep, that works great. Thanks for a quick fix!

**Comment by pacak** [comment]
Released 0.1.18
